app/src/main/java/com/pajato/android/gamechat/chat/fragment/ShowGroupListFragment.java

```diff
@@ -52,9 +53,10 @@
     public static final String CHAT_GROUP_FAM_KEY = "chatGroupFamKey";
 
     // Public instance methods.
-
-    /** Set the layout file. */
-    @Override public int getLayout() {return R.layout.fragment_chat_list;}
+    @Override public void onCreate(Bundle bundle) {
```

Comment:
In this case the existing doc comment is pretty lame.  Please add back one that is less lame.